java-strings/src/test/java/com/baeldung/string/WordCountUnitTest.java

```diff
@@ -0,0 +1,57 @@
+package com.baeldung.string;
+
+import static org.junit.Assert.*;
+
+import org.junit.Test;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+/**
+ *
+ * @author shwetaJ
+ */
+
+public class WordCountUnitTest {
+	private static final Logger logger = LoggerFactory.getLogger(WordCountUnitTest.class);
+
+	String stringIn = "I like  Cookies, Cake & Icecream but not chocolates! ";
+
+	String stringTocheck = stringIn.trim().replaceAll(" +", " ");
+
+	@Test
+	public void testCountUsingforLoop() {
+		int count = WordCount.countUsingCharAt(stringTocheck, ' ');
+		assertEquals(count, 9);
```

Comment:
The assert equals should have the expected value first. So it should be `assertEquals(0, count)`

Reply:
Thanks for the feedback.I 'll do the changes, as suggested by you.